renderer/plugin.js

```diff
@@ -38,14 +43,27 @@ org.ekstep.questionunitseq.RendererPlugin = org.ekstep.contentrenderer.questionU
     var correctAnswer = true;
     var correctAnswersCount = 0;
     var telemetryValues = [];
+    var seq_rearranged = [];
     var totalOptions = instance._question.data.options.length;
+
+    for(var i = 0;i < totalOptions;i++){
```

Comment:
Why do you need this logic. seq_rearranged = [].
You can handle this in the below code changes